rosdep/python.yaml

```diff
@@ -4101,6 +4101,7 @@ python-serial:
     artful_python3: [python3-serial]
     bionic: [python-serial]
     bionic_python3: [python3-serial]
+    focal: [python3-serial]
```

Comment:
We should not be using python3-serial for a python-serial key. It should be in a new python3-serial key.

Also instead of adding just focal we should use `'*': [python-serial]` and just leave the legacy `_python3` version rules.

Reply:
Thanks for the comment, I didn't check that there was already a key for python3-serial, my bad.

I've added the `focal_python3` entry instead and created `'*': [python-serial]` as requested